tests/console/system_prepare.pm

```diff
@@ -68,6 +68,15 @@ sub run {
         }
     }
 
+    # This workaround is intentionally guarded for Tumbleweed only - SLE must never ever accept that as a workaround
+    if (is_tumbleweed) {
```

Comment:
+1 for targeting only Tumbleweed.

`git grep /etc/nsswitch.conf` in openQA code shows several different approaches. Some code correctly looks for both `/usr/etc/nsswitch.conf` and `/etc/nsswitch.conf`. But there are at least 2 cases when `/etc/nsswitch.conf` is being created:
```
data/sssd-tests/testincl.sh:  cp /usr/etc/nsswitch.conf /etc/nsswitch.conf
tests/console/systemd_resolved.pm:    assert_script_run "cp /usr/etc/nsswitch.conf /etc/nsswitch.conf";
```

Reply:
those two tests intentionally copy as they are modifying the file afterwards; this is exactly the correct way to do it